### DISCUSSION TITLE
Hi @s-madhavan001 👋🏻 All you need is here: https://github.com/roboflow/supervision/tree/develop/examples/time_in_zone

### DISCUSSION THREAD

**Discussion by s-madhavan001** [roboflow/supervision#1095] [Q&A]
Hi @s-madhavan001 👋🏻 All you need is here: https://github.com/roboflow/supervision/tree/develop/examples/time_in_zone

Originally posted by @SkalskiP in https://github.com/roboflow/supervision/discussions/950#discussioncomment-8952710

i want to detect employs and all others also but i want to calculate customer waiting time only same like that?

**Comment by LinasKo** [comment]
Hi @s-madhavan001,

I've seen there's a new video out. Perhaps you'd find it useful?
https://www.youtube.com/watch?v=hAWpsIuem10

**Comment by SkalskiP** [comment]
Hi @s-madhavan001 👋🏻 yesterday, we released a YouTube tutorial explaining the whole time-in-zone demo. https://www.youtube.com/watch?v=hAWpsIuem10

I hope it will answer all your questions. I'm converting this issue into a discussion and placing it in the Q&A section.
